vimdoc/regex.py

```diff
@@ -211,7 +215,8 @@ def _DelimitedRegex(pattern):
   # MATCH GROUP 1: The Name
   (
     # Non-commas or escaped commas or escaped escapes.
-    (?:[^\\,]|\\.)+
+    # Must not end with a space.
```

Comment:
valuable in itself - it prevents problems with trailing spaces being changed to dashes.